code/modules/clothing/modular_armor/attachments.dm

```diff
@@ -253,3 +278,8 @@
 /obj/item/armor_module/armor/proc/extra_examine(datum/source, mob/user, list/examine_list)
 	SIGNAL_HANDLER
 	examine_list += "Right click the [parent] with paint to color the [src]"
+
+/obj/item/armor_module/armor/on_vend(faction)
+	..()
```

Comment:
Always keep the return value of ..() in .

Reply:
Preserved return values with . = ..() for all calls that did not do so.